digdag-standards/src/main/java/io/digdag/standards/operator/HttpCallOperatorFactory.java

```diff
@@ -133,6 +133,8 @@ public TaskResult runTask()
 
             // parse content based on response media type
             String digFileSource = reformatDigFile(content, response.getMediaType(), mediaTypeOverride);
+            // write workflow definition to log
+            logger.info("Fetched definition for http_call>:\n{}", digFileSource);
```

Comment:
I am afraid of increasing server log.  Current max_stored_response_content_size is 262KB.
So this logging record is near to 262KB in maximum.

Reply:
@yoyama Thank you for comment.

Even if `config.http_call.max_response_content_size` is too large, I assume log string size could not be a problem, because the log string will be out through `TaskLogger` and its implementations have [some mechanisms to handle huge amount of log strings](https://github.com/treasure-data/digdag/blob/d3fe54b9dfa3a924fceafd31febbdc25851560b8/digdag-core/src/main/java/io/digdag/core/log/BufferedRemoteTaskLogger.java#L63-L65).
In addition, other factories possible to write large volume logs, for instance, `TdOperatorFactory` [writes query statement](https://github.com/treasure-data/digdag/blob/7309575d66fe297e91edf35dce4cba094ecf2a8c/digdag-standards/src/main/java/io/digdag/standards/operator/td/TdOperatorFactory.java#L236) that may be large.